docs/python/how-to-guides/use-uris.md

```diff
@@ -119,16 +202,45 @@ If we have the hostname of our colleague's machine, that can be used in place of
 
 If the machine on which a table exists is public, then consuming that table is done the same way as if it were a private network. All that's needed is the hostname/IP and table name.
 
-<!-- TODO:
+## Performance considerations
+
+When using URIs to share tables across instances, particularly over networks, there are several performance factors to consider:
+
+### Network impact
+
+- **Latency**: Table access over a network introduces latency that varies based on network conditions. For operations requiring low latency, consider co-locating instances when possible.
+- **Bandwidth**: Large tables or tables with frequent updates require more bandwidth. Network throughput can become a bottleneck for data-intensive operations.
+- **Connection reliability**: Unstable network connections can affect the reliability of table access. Implement appropriate error handling for network disruptions.
+
+### Table characteristics
+
+- **Table size**: Larger tables require more resources to transfer and update. The overhead increases with row count and column width.
```

Comment:
This point assumes that URI sharing transmits entire tables over a network/across instances. Is this the case, or does it only share some of the data? I'm not 100% sure how URI sharing actually shares tables.